### PR TITLE
Speed up trials

### PR DESCRIPTION
**Pull request by joakimmoller**
logging configuration
gunicorn Preload app
refactor routes to blueprints
connection pooling configured (sqlalchemy)
file_info endpoint using sqlalchemy pool

### CODE REVIEW

File: src/odinapi/api.py [hunk 1]
```diff
@@ -1,463 +1,26 @@
-"""A simple datamodel implementation"""
+"""A complex datamodel implementation"""
```
Comment (skymandr) [inline src/odinapi/api.py:1]: :rofl:

File: src/odinapi/run.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from .blueprints import register_blueprints
+from .api import app
+
+register_blueprints(app)
+if __name__ == "__main__":
+    app.run()
```
Comment (skymandr) [inline src/odinapi/run.py:6]:
```suggestion
    app.run()

```

File: src/odinapi/views/views.py [hunk 3]
```diff
@@ -38,7 +38,8 @@
 
 # Make linter happy
 use_agg
-
+SWAGGER.add_parameter("freqmode", "path", int)
+SWAGGER.add_parameter("scanno", "path", int)
 SWAGGER.add_parameter('date', 'path', str, string_format='date')
 SWAGGER.add_type('freqmode_info', {
     'Backend': str,
```
Comment (skymandr) [inline src/odinapi/views/views.py:45]: Have you checked how the swagger docs renders with your changes?

**Review by skymandr — APPROVED** [review]
:rocket:

File: src/odinapi/views/views.py (hunk 3 above)
Comment (joakimmoller) [reply src/odinapi/views/views.py:45]: No, but there are a lot of add_parameter calls scattered around the code  so i don't think order matters. Probably we should just cut the swagger-try-to-integrate-the-code and replace with a standard openapi yaml file at some point...

File: src/odinapi/run.py (hunk 2 above)
Comment (joakimmoller) [reply src/odinapi/run.py:6]: Cannot se what change you are requesting. The idea here is to make it possible to start flask with run from the command line, but when gunicorn starts, it just uses the app object.

**Pull request merged**